vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -466,6 +467,11 @@
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\textlayoutmanager\TextMeasureCache.cpp" />
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\uimanager\UIManager.cpp" />
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\uimanager\UIManagerBinding.cpp" DisableSpecificWarnings="4389;4715;%(DisableSpecificWarnings)" />
+    <ClCompile Include="$(ReactNativeWindowsDir)codegen\react\components\rnwcore\Props.cpp" >
+      <ObjectFileName>$(IntDir)\codegenRnwCoreProps.obj</ObjectFileName>
```

Comment:
OOC, why do we change the object file output for this translation unit?

Reply:
It doesn't like having two cpp files with the same name.  -- There is another Props.cpp in "$(ReactNativeDir)\ReactCommon\react\renderer\core\Props.cpp"